src/sonic-host-services/tests/procdockerstatsd_test.py

```diff
@@ -36,9 +36,14 @@ def test_convert_to_bytes(self):
             ('500MiB', 524288000),
             ('66.41MiB', 69635932),
             ('333.6MiB', 349804954),
+            ('7.75GB', 7750000000),
+            ('800GB', 800000000000),
             ('1GiB', 1073741824),
             ('500GiB', 536870912000),
-            ('7.751GiB', 8322572878)
+            ('7.751GiB', 8322572878),
+            ('8.1TiB', 8906044184986),
+            ('12TB', 12000000000000),
+            ('2.25TB', 2250000000000)
```

Comment:
the test is too simple, we need to have a test that convert from the ps -e output and test the content in the database as end-to-end test.

Reply:
I think this test is appropriate to the `convert_to_bytes()` method. More test cases need to be added in order to ensure full end-to-end functionality.